ui/desktop/electron-app/src/index.js

```diff
@@ -142,12 +142,12 @@ const createWindow = (partition, closeWindowCB) => {
   });
 
   // Opens external links in the host default browser.
-  // We allow boundaryproject.io domain to open on external window (for now)
+  // We allow developer.hashicorp.com domain to open on external window
   // and releases.hashicorp.com domain to download the desktop app or
   // link to the release page for the desktop app.
   browserWindow.webContents.setWindowOpenHandler(({ url }) => {
     if (
-      url.startsWith('https://boundaryproject.io/') ||
+      url.startsWith('https://developer.hashicorp.com/boundary/') ||
```

Comment:
Do we need to be specific for `/boundary`?

Also do we not have any links at all to `boundaryproject.io` anymore?

Reply:
All the links we have are in the `/boundary` path so thats why i included it in the base url but it does not need to be like that. Would it be better if we leave out the `boundary` path and add it to the individual path mappings instead in case theres a future scenario where we need to link to a doc that's not in `/boundary`? If so, i agree with that line of reasoning and can update the paths.

Nope i as i was updating i checked that all the links redirect to the developer docs.

Reply:
I was just a little worried because it's an easy gotcha in the future which we might not notice.  I also think there's no reason we can't trust the hashicorp base domain rather than the specific `/boundary` path and my vote would be to just leave it out